Dockerfile

```
@@ -153,6 +152,21 @@ RUN set -eux; \
         done; \
     done
 
+# Add a couple 3rd party extension managers to make extension additions easier
+RUN set -eux; \
+    apt-get install -y pgxnclient; \
+    cargo install pg-trunk
+
+RUN set -eux; \
+    for pg in ${PG_VERSIONS}; do \
+        for pkg in h3 orafce pgTAP; do \
+            pgxn install --pg_config "/usr/lib/postgresql/${pg}/bin/pg_config" "$pkg"; \
+        done; \
```

Both `orafce` and `pgtap` exist in the pgdg debian repos, so we don't need to install them here like this

```suggestion
        for pkg in h3; do \
            pgxn install --pg_config "/usr/lib/postgresql/${pg}/bin/pg_config" "$pkg"; \
        done; \
```